### DISCUSSION TITLE
Attempting to add Fugitive, seemingly nothing happens.

### DISCUSSION THREAD

**Discussion by jsatk** [LazyVim/LazyVim#1508] [Q&A] [answered]
Following the instructions here: https://www.lazyvim.org/configuration/plugins.

In `~/.config/nvim/lua/plugins/fugitive.lua` I have the following:

```lua
return {
  { "tpope/vim-fugitive" },
}
```

Upon restarting nvim and opening Lazy it is not there, no prompt to install, not in list. My `config/lazy.lua` is borderline default.

```lua
require("lazy").setup({
  spec = {
    -- add LazyVim and import its plugins
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },

    -- import any extras modules here
    { import = "lazyvim.plugins.extras.dap.core" },
    { import = "lazyvim.plugins.extras.lang.typescript" },
    { import = "lazyvim.plugins.extras.lang.rust" },
    { import = "lazyvim.plugins.extras.lang.docker" },
    { import = "lazyvim.plugins.extras.coding.copilot" },

    -- import/override with your plugins
    { import = "plugins" },
  },
  ...
```

Any idea how to get this plugin loading?

**Comment by abeldekat** [comment]
Hello @jsatk,

When I apply your code to stock lazyvim starter, the plugin loads just fine.
Perhaps you changed lazy.nvim's opts in `config.lazy`, setting `defaults.lazy = true`?

**Reply by jsatk** [reply]
It's still set to the defaut:

```lua
  defaults = {
    -- By default, only LazyVim plugins will be lazy-loaded. Your custom plugins will load during startup.
    -- If you know what you're doing, you can set this to `true` to have all your custom plugins lazy-loaded by default.
    lazy = false,
```

**Reply by jsatk** [reply, marked as answer]
OMG 🤦  It's because I did not run `rcup` which symlinks files from my `~/.dotfiles` directory to their appropriate spots.
